test/e2e/README.md

```diff
@@ -30,26 +30,24 @@ knative-gcp should be added under [knative-gcp e2e test lib](lib).
 
 There are two ways to set up authentication mechanism.
 
-- If you want to run E2E tests with authentication mechanism using
-  **[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)**,
-  please follow below instructions to configure the authentication mechanism
-  with **[Workload
-  Identity]**(https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
-  **[Workload
-  Identity]**(https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
-  is GKE specific.
-- If you want to run E2E tests with authentication mechanism using **Kubernetes
-  Secrets**, please follow below instructions to configure the authentication
-  mechanism with **Kubernetes Secrets**.
+- (GKE Specific) If you want to run E2E tests with **[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)**
+  as the authentication mechanism, please follow below instructions to configure the authentication mechanism
+  with **[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)**.
+- If you want to run E2E tests with **Kubernetes Secrets** as the authentication mechanism, please
+  follow below instructions to configure the authentication mechanism with **Kubernetes Secrets**.
 
 1.  A running Kubernetes cluster with
     [knative-gcp](../../docs/install/install-knative-gcp.md) installed and
     configured.
-1.  [Pub/Sub Enabled Service Account](../../docs/install/pubsub-service-account.md)
-    installed.
-1.  [GCP Broker Deployment](../../docs/install/install-gcp-broker.md#deployment)
-    and
-    [GCP Broker Authentication Setup](../../docs/install/install-gcp-broker.md#authentication-setup-for-gcp-broker).
+1.  Create a [Pub/Sub Enabled Service Account](../../docs/install/pubsub-service-account.md).
+    Download a credential file and set `GOOGLE_APPLICATION_CREDENTIALS` env var. This is used by
+    some tests(e.g., `TestSmokePullSubscription`) to authorize the Google SDK clients.
+    ```
+    gcloud iam service-accounts keys create cre-pubsub.json \
+    --iam-account=cre-pubsub@$PROJECT_ID.iam.gserviceaccount.com
+    export GOOGLE_APPLICATION_CREDENTIALS=./cre-pubsub.json
+    ```
+1.  [Install GCP Broker](../../docs/install/install-gcp-broker.md#deployment).
```

Comment:
Is it a little bit confusing here if we don't say it clearly that only deployment and authentication set up are needed?
People may forget to do the authentication set up since you only link to the `deployment` session or people may think they have to create a GCP Broker by themselves.

Reply:
As part of the deployment, we ask user to set up authentication. It's now the first step in the deployment guide.

Reply:
In this case, why the link is ../../docs/install/install-gcp-broker.md#deployment instead of https://github.com/liu-cong/knative-gcp/blob/docs-new/docs/install/install-gcp-broker.md?

Reply:
oops, updated link